internal/git/commands_test.go

```diff
@@ -17,6 +17,7 @@ import (
 func TestBackendCommands(t *testing.T) {
 	t.Parallel()
 	initial := gitdomain.NewLocalBranchName("initial")
+	// Keep tests sorted.
```

Comment:
Apologies for asking you to revise this again. Most files in this codebase are sorted alphabetically. How do you feel about moving this comment to the developer documentation in [docs/DEVELOPMENT.md](https://github.com/git-town/git-town/blob/main/docs/DEVELOPMENT.md)? Placing it there makes it more visible and improves the onboarding experience for all new contributors.

Reply:
Actually, it would fit very well into https://github.com/git-town/git-town/blob/main/docs/ARCHITECTURE.md#code-style.

Reply:
If you don't feel like writing so much documentation, I'm happy to do it in a separate pull request. Just let me know.

Reply:
While reviewing the architecture docs, I noticed a few unrelated opportunities for improvement and made some quick updates, including relocating your comment from this PR. I'm merging this PR now. Great work!

Reply:
Ah, cool, no worries. I don't like documenting a lot of stuff as bigger docs are less read. Instead I was thinking to add a linter: https://github.com/git-town/git-town/pull/4403. This would ensure that the style is automatically enforced.